qiskit/synthesis/evolution/suzuki_trotter.py

```diff
@@ -105,51 +108,91 @@ def __init__(
             ValueError: If order is not even
         """
 
-        if order % 2 == 1:
+        if order > 1 and order % 2 == 1:
             raise ValueError(
                 "Suzuki product formulae are symmetric and therefore only defined "
-                "for even orders."
+                "for even orders (or order==1)."
```

Comment:
Can we slightly improve this error message, maybe to "Suzuki product formulas are only defined when the order is 1 or even"?

Reply:
Updated in e802c45